Pipeline/services/Vernal.py

```diff
@@ -2,9 +2,14 @@
 # it'll only be pure http call, as in send the inputs and receive the outputs
 # manipulation of that data should be done elsewhere
 import requests 
-URL = 'http://vernal:5003/'
+URL = "http://0.0.0.0:5003/"
 
-def greeting():
-    r = requests.get(url = URL) 
-    data = r.json() 
-    return data
+def vernalSimilarityFunction(representative_graphs, dataset):
+
+    data = {"graphs": str(representative_graphs), "dataset": dataset}
+    headers = {}
+    files = []
+    response = requests.request(
+        "POST", URL + "CompareSequence/", headers=headers, data=data, files=files)
```

Comment:
I told you in your [vernal review](https://github.com/TeamW-P/VeRNAl/pull/6#discussion_r589170078) to change url path name 😤😤 lol

but friendly reminder that it should be `/compare-sequence` so like change to lowercase, add hyphen and lose the trailing slash

 you should change it here and in VeRNAl

Reply:
changes made